analysis/src/oxDNA_analysis_tools/config.py

```diff
@@ -46,7 +46,7 @@ def check(to_check:List[str]=["python", "numpy", "matplotlib", "sklearn", "oxpy"
         print("INFO: Python version: {}".format('.'.join([str(i) for i in version_info[0:3]])), file=stderr)
         if version_info < (3, 9):
             flag = True
-            print("WARNING: Some scripts will not run with Python versions earler than 3.8.  You have {}, please update your environment", file=stderr)
+            print("WARNING: Some scripts will not run with Python versions earler than 3.9.  You have {}, please update your environment", file=stderr)
```

Comment:
I think there is a `version_info` missing in the print call

Reply:
And there is also a typo (earler should be earlier)

Reply:
As far as version requirements go, the oxpy/oat code in this pull request will work with Python 3.9-3.11 (haven't checked the bleeding edge 3.12).  Because of changes to the CPython api, the current version of Pybind11 (2.11) only works with Python 3.8+.  Because of some data structures and type hints in oat, it needs 3.9+.  

I'm not familiar with CMake/make enough to do this easily, but is there an easy way to decouple oxpy from oat (since right now they're just installed in a single pip command, I think)?  If so, we could give instructions on how to install just oxpy with older versions of Python by downloading an older version of Pybind11 and replacing the `oxpy/pybind11` folder.

Thank you for noticing the errors.  I will fix.

Reply:
I added a section to the Installation page of the documentation on how to replace the pybind11 folder and remove oat, which will allow oxpy to be installed with older Python versions.